src/ert/_c_wrappers/config/config_parser.py

```diff
@@ -26,6 +53,68 @@ def __init__(self, errors: str, config_file: Optional[str] = None) -> None:
             else f"{self.errors}"
         )
 
+    def replace(self, old_text: str, new_text: str):
+        return ConfigValidationError(
+            errors=self.errors.replace(old_text, new_text),
+            config_file=self.config_file,
+            location=self.location,
+        )
+
+    @property
+    def config_file(self):
+        return self.location.filename
+
+    @config_file.setter
+    def config_file(self, config_file):
+        self.location.filename = config_file
+
+    def get_error_messages(self):
+        return [self.errors]
+
+
+class CombinedConfigError(ConfigValidationError):
+    def __init__(
+        self,
+        errors: Optional[
+            List[Union[ConfigValidationError, "CombinedConfigError"]]
+        ] = None,
+    ):
+        self.errors = []
+
+        for err in errors or []:
+            self.add_error(err)
+
+    def __str__(self):
+        return ", ".join(str(x) for x in self.errors)
+
+    def is_empty(self):
+        return len(self.errors) == 0
+
+    def add_error(self, error: Union[ConfigValidationError, "CombinedConfigError"]):
+        if isinstance(error, CombinedConfigError):
+            self.errors.append(*error.errors)
```

Comment:
`self.errors.append(*error.errors)` crashes with `list.append() takes exactly one argument`. You need to do `self.errors += error.errors`

Reply:
Seems like we may have to create a test that hits this condition.